### ISSUE TITLE
Shouldn't this be vkSS::eClosestHitKHR | vkSS::eAnyHitKHR instead? 

### ISSUE THREAD

**Issue by BattleAxeVR** [nvpro-samples/vk_raytracing_tutorial_KHR#8]
https://github.com/nvpro-samples/vk_raytracing_tutorial_KHR/blob/81c26d451fa7daefd585baf27433cbc1b65fa0dc/ray_tracing_gltf/hello_vulkan.cpp#L694

**Comment by mklefrancois** [comment]
Yes, thanks, I will fix this in the next update. Fortunately, they are aliased with NV.

**Comment by BattleAxeVR** [comment]
Yeah I figured, because the samples work with or without the fix locally. I'd do a sweep on the rest to make sure they're correct.
Great repo BTW. 

I modded the entire gltf sample to use base C Vulkan API because I don't like the HPP wrapper stuff, to my mind it's obfuscating parameters in extra layers of indirection for little benefit. But I've learned a lot (and found issues like these) by doing that. If I find any more I'll post'em here. There are a few other suggestions I'd make like renaming the graphics pipeline member vars vs postprocess pipeline explicitly, and not making duplicates of pointers by passing them by reference to their constructor instead of the various "setup" functions. But that's neither here nor there. I just think duplicated raw pointers are an anti-pattern in C++ land
